packages/firestore/src/local/shared_client_state_schema.ts

```diff
@@ -115,6 +115,18 @@ export function createWebStorageOnlineStateKey(persistenceKey: string): string {
   return `${ONLINE_STATE_KEY_PREFIX}_${persistenceKey}`;
 }
 
+// The WebStorage prefix that plays as a event to indicate the remote documents
+// might have changed due to some actions secondary tabs did.
+// format of the key is:
+//     firestore_remote_documents_changed_<persistence_prefix>
+export const REMOTE_DOCUMENTS_CHANGED_KEY_PREFIX =
+  'firestore_remote_documents_changed';
```

Comment:
So the remote documents change when queries get updated and even when mutations are committed, but this even only fires when a bundle is loaded. Should this be a bundle specific notification?

Reply:
Please also update the key prefix. For sake of brevity, you could consider dropping "remote documents" altogether and just make it something like "bundle loaded".

Reply:
Done.